packages/astro/astro.js

```diff
@@ -46,7 +46,7 @@ async function main() {
   // it's okay to hard-code the valid Node versions here since they will not change over time.
   if (typeof require === 'undefined') {
     console.error(`\nNode.js v${version} is not supported by Astro!
-Please upgrade to a version of Node.js with complete ESM support: "^12.20.0 || ^14.13.1 || >=16.0.0"\n`);
+Please upgrade to a version of Node.js with complete ESM support: "^14.15.0 || >=16.0.0"\n`);
```

Comment:
Not required in this PR, but we may now be able to remove `supports-esm` and most of the logic in this file and just check that `process.versions.node > '14.15'` instead of each ESM feature individually.

The context is that most of these checks were needed because Node v12 changed from version-to-version, and Node v14 was still actively supported and undergoing changes from version-to-version. Now it should be safe to do the more general ` > '14.15'` check and leave it at that.